src/cdk/v2/destinations/bingads_audience/procWorkflow.yaml

```diff
@@ -0,0 +1,50 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - path: ../../bindings/jsontemplate
+    exportAll: true
+  - path: ./config
+  - name: removeUndefinedAndNullValues
+    path: ../../../../v0/util
+  - name: defaultRequestConfig
+    path: ../../../../v0/util
+
+steps:
+  - name: validateInput
+    template: |
+      let messageType = .message.type;
+      $.assert(.message.type, "message Type is not present. Aborting message.");
+      $.assert(.message.type.toLowerCase(), "Event type " + .message.type.toLowerCase() + " is not supported. Aborting message.");
```

Comment:
Curious question(Non-blocking)
What would happen when `message.type` is sent as a number ?

Reply:
It will throw error as lowercase will not work on that. Should we handle this, generally we don't handle this, as is not expected.

Reply:
Ideally yes. But probably event-type being a number would not be a valid-case at all. So this is ok. Hence resolving the comment